### PR TITLE
Apps: Decorate the smartphone apps articles, issue #200

### PR DESCRIPTION
**Pull request by DocFraggle**
Apps: Decorate the smartphone apps articles, issue #200 
Android part

### CODE REVIEW

File: addons/uis/apps/android.md [hunk 1]
```diff
@@ -12,14 +12,17 @@ It uses the REST API of openHAB to render sitemaps of your openHAB installation.
...
-<a href="https://play.google.com/store/apps/details?id=org.openhab.habdroid">
-  <img alt="'Get the openHAB app on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" style="width: 50%; height: 50%"/>
-</a>
+* Username/password or SSL client authentication
+* Definition of default sitemap
+* multiple themes
+* Thing discovery via app
+* Voice commands
+
+[![Download openHAB app on Google Play](images/google-play-badge.png){:width="250px"}](https://play.google.com/store/apps/details?id=org.openhab.habdroid)
```
Comment (ThomDietrich) [inline addons/uis/apps/android.md]: Did you check, that this change is indeed working?

File: addons/uis/apps/android.md [hunk 2]
```diff
@@ -12,14 +12,17 @@ It uses the REST API of openHAB to render sitemaps of your openHAB installation.
...
+
+![](images/android_01.png){:width="250px"} ![](images/android_02.png){:width="250px"}
```
Comment (ThomDietrich) [inline addons/uis/apps/android.md]: better 50% ?

File: addons/uis/apps/android.md [hunk 3]
```diff
@@ -12,14 +12,17 @@ It uses the REST API of openHAB to render sitemaps of your openHAB installation.
...
-[![Download openHAB app on Google Play](images/google-play-badge.png)](https://play.google.com/store/apps/details?id=org.openhab.habdroid)
--->
+**Features:**
 
-<a href="https://play.google.com/store/apps/details?id=org.openhab.habdroid">
-  <img alt="'Get the openHAB app on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" style="width: 50%; height: 50%"/>
-</a>
+* Username/password or SSL client authentication
+* Definition of default sitemap
+* multiple themes
```
Comment (ThomDietrich) [inline addons/uis/apps/android.md]: "Multiple themes"

**Comment by ThomDietrich** [comment]
Hey, two more requests:

1. Would you please add a 1-2 general features? I'm missing something like "View openHAB sitemaps" before starting to talk about SSL ;)
2. You need to sign all your commits: https://github.com/openhab/openhab-docs/blob/gh-pages/CONTRIBUTING.md

**Comment by ThomDietrich** [comment]
One more thing. Something seems to be wrong with the PlayStore image. Compare
* https://github.com/openhab/openhab-docs/blob/gh-pages/addons/uis/apps/images/download-on-the-app-store.png
* https://github.com/openhab/openhab-docs/blob/gh-pages/addons/uis/apps/images/google-play-badge.png

Could you please replace the file `google-play-badge.png` by the one downloadable here: https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png

**Comment by DocFraggle** [comment]
Hi Thomas,

general features added and signed. The images are exactly the same, what am I missing here? :)

Regards, Christian

**Comment by ThomDietrich** [comment]
You are absolutely right. Must be something related to caching or similar on my system. I can't look at the image, it's getting rendered as one small black 1x1px.

Nevermind.

**Comment by ThomDietrich** [comment]
Thank you!

**Pull request merged**